### ISSUE TITLE
Account unable to skip AD directories to which I do not have access.

### ISSUE THREAD

**Issue by jcordtz** [microsoft/AzureStorageExplorer#4014]
**Storage Explorer Version**: 1.17.0
**Build Number**: 20201211.10
**Platform/OS**: Windows10
**Architecture**: x64
**Regression From**:  No

## Bug Description ##
When I login in to my Azure account storage explorer will pick up all directories to which I am actually connected. Some of them are directories external to Microsoft. Therefor Storage Explorer tries to make login, but fails, because I do not have that access any longer (but I am not completely removed from the directories in question). Storage Explorer repeatedly tries the login and only after cancel "many" of those I end up in the subscriptions I really have access. But the issue is that my account stays in "login as to be fixed", so I do not get a list of subscriptions I can choose from. This makes storage explorer kind of irritating to work with because I am not in control of which subscriptions actually is of interest to my work.  

## Steps to Reproduce ##
Add account simply fails in the login process when encountering a directory failing.

## Expected Experience ##
That I get a list of subscriptions under my account I actually have access to and can choose from.

## Actual Experience ##
A very messy Storage Explorer experience me being out of control

## Additional Context ##
None
![expl1](https://user-images.githubusercontent.com/17173791/105854165-d5dbca00-5fe6-11eb-80a5-49f887f7756b.jpg)
![expl2](https://user-images.githubusercontent.com/17173791/105854178-db391480-5fe6-11eb-811f-0f4fbabcf06f.jpg)

**Comment by MRayermannMSFT** [comment]
Duplicate of #460
